app/Http/Controllers/Api/ArtistController.php

```diff
@@ -30,10 +30,10 @@ public function __construct(ArtistRepositoryInterface $artist)
      *
      * @return \Illuminate\Http\Resources\Json\AnonymousResourceCollection
      */
-    public function index()
+    public function all()
```

Comment:
Glad you caught this. I had meant to ask you if we should change the ones I had done; I like `all()` better, too.